docs/source/apis/product.rst

```diff
@@ -0,0 +1,402 @@
+  
+   `GetProductCategoriesForSKU <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getproductcategoriesforsku>`_ = `get_product_categories_for_sku() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L224>`_
+  
+   `GetProductCategoriesForASIN <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getproductcategoriesforasin>`_ = `get_product_categories_for_asin() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L236>`_
+  
+  
+***************
+Getting Started
+***************
+  
+Please read through gettingStarted.rst, as these docs build upon those.
+
+.. code-block:: python
+
+      import mws
+
+      access_key = “XXXXXXXXXXXXXXXXXXXX”
+      secret_key = “XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX”
+      seller_id = “XXXXXXXXXXXXXX”
+      region = “XX”
+      my_marketplace = Marketplace.US.marketplace_id
+      
+      products_api = mws.Products(access_key, secret_key, seller_id, region)
+  
+  
+  
+************************
+`ListMatchingProducts <https://docs.developer.amazonservices.com/en_US/products/Products_ListMatchingProducts.html>`_
+************************
+
+From Amazon:
+
+   The ListMatchingProducts operation returns a list of products and their attributes, ordered by relevancy, based on a search query that you specify. Your search query can be a phrase that describes the product or it can be a product identifier such as a GCID, UPC, EAN, ISBN, or JAN. Returns 10 results.
+  
+
+.. code-block:: python
+
+      response = products_api.list_matching_products(
+         marketplace_id = my_marketplace,
+         query = “Amazon alexa”
+      )
+
+
+
+  
+Examples
+========
+
+We can access individual results with:
+
+.. code-block:: python
+
+      response.parsed.Products.product[0]
+  
+Sales Rank:
+
+.. code-block:: python
+
+      response.parsed.Products.Product[0].SalesRankings.SalesRank[0].Rank
+      # 6
+  
+ASIN:
+
+.. code-block:: python
+
+      response.parsed.Products.Product[0].Identifiers.MarketplaceASIN.ASIN
+      # B085G58KWT
+  
+Title:
+
+.. code-block:: python
+
+      response.parsed.Products.Product[0].AttributeSets.ItemAttributes.Title
+      # All-new Echo (4th generation) | With premium sound, smart home hub and Alexa | Charcoal
+  
+  
+
+*********************
+`GetMatchingProduct <http://docs.developer.amazonservices.com/en_US/products/Products_GetMatchingProduct.html>`_
+*********************
+
+From Amazon:
+   The GetMatchingProduct operation returns a list of products and their attributes, based on a list of ASIN values that you specify. This operation returns a maximum of ten products.
+  
+Can supply ASINs as a list:
+.. code-block:: python
+
+      response = products_api.get_matching_product(
```

Comment:
from memory the limit of asin's to be sent is 20 - could you check and add this into the docs?